simavr/sim/avr_spi.h

```diff
@@ -103,4 +104,6 @@ void avr_spi_init(avr_t * avr, avr_spi_t * port);
 };
 #endif
 
+static const uint8_t _avr_spi_clkdiv[4] = {4,16,64,128};
+
```

Comment:
This should definitely go in the .c if you don't mind, or even in the function/block that uses it!

Reply:
Sure thing.